src/main/scala/nodes/learning/KernelModel.scala

```diff
@@ -0,0 +1,78 @@
+package nodes.learning
+
+import breeze.linalg._
+import edu.berkeley.cs.amplab.mlmatrix.{RowPartition, NormalEquations, BlockCoordinateDescent, RowPartitionedMatrix}
+import nodes.stats.{StandardScalerModel, StandardScaler}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.broadcast.Broadcast
+import nodes.util.{VectorSplitter, Identity}
+import utils.{MatrixUtils, Stats, KernelUtils}
+import workflow.{Transformer, LabelEstimator}
+
+
+/**
+ * @param models different kernel models for different lambdas
+ * @param blockSize blockSize to split data before applying transformations
+ * @param lambdas  lambdas corresponding to the models
+ * @param trainData the training Data
+ * @param kernelGen the kernel generator
+ **/
+
+class KernelBlockModel(
+  models: Broadcast[Array[DenseMatrix[Double]]],
```

Comment:
Would rather not have a `Broadcast` here - if we get some new RDD with some new `SparkContext` we should hopefully be able to apply the model there (though maybe not if we really need `TrainData` below). 

I guess it would be good to have some utilities for loading one of these given a SparkContext and a pointer to the `TrainData` on HDFS.